.devcontainer/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM golang:1.18 AS go-build
+RUN go install github.com/apache/skywalking-eyes/cmd/license-eye@latest
```

Comment:
How about removing this? We can rely on CI to check this (I didn't have license-eye locally too...)

Reply:
emmm, it doesn't affect build time much, i can remove it if you want to.

Reply:
Let's remove it

Reply:
done